### PR TITLE
follow up server info

### PR DESCRIPTION
**Pull request by jitendrajaria**

### CODE REVIEW

File: src/main/java/creativei/enums/CaseIndex.java [hunk 1]
```diff
@@ -18,4 +25,28 @@ public int getIndex() {
...
+    public void setValue(String value) {
+        this.value = value;
+    }
+    static List<String> list = new ArrayList<>();
+    static {
+        for (CaseIndex caseIndex : CaseIndex.values()) {
+            try {
+                list.add(new ObjectMapper().writeValueAsString(caseIndex));
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
```
Comment (aayushKhandpur) [inline src/main/java/creativei/enums/CaseIndex.java]: Use logger

File: src/main/java/creativei/enums/FollowUpSubStatus.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package creativei.enums;
 
+import java.util.List;
+import java.util.stream.*;
+
 public enum FollowUpSubStatus {
     MORE_COUNSELLING("Need More Cunselling"),
```
Comment (aayushKhandpur) [inline src/main/java/creativei/enums/FollowUpSubStatus.java]: Type - Counselling

File: src/main/java/creativei/enums/FollowUpSubStatus.java [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
     ENROLLMENT_IN_THIRTY_DAYS("Possible Enrollment In A Month"),
```
Comment (aayushKhandpur) [inline src/main/java/creativei/enums/FollowUpSubStatus.java]: no need for capitalized words in value field.- Eg- Possible enrollment in a month

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review comments.

File: src/main/java/creativei/enums/FollowUpSubStatus.java (hunk 3 above)
Comment (jitendrajaria) [reply src/main/java/creativei/enums/FollowUpSubStatus.java]: should i change all of them or should i change only Possible enrollment in a month

File: src/main/java/creativei/enums/CaseIndex.java (hunk 1 above)
Comment (jitendra20) [reply src/main/java/creativei/enums/CaseIndex.java]: added in commit 680b8b1

File: src/main/java/creativei/enums/FollowUpSubStatus.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/enums/FollowUpSubStatus.java]: resolved in commit  680b8b1

File: src/main/java/creativei/enums/FollowUpSubStatus.java (hunk 3 above)
Comment (jitendrajaria) [reply src/main/java/creativei/enums/FollowUpSubStatus.java]: issue resolved in commit 680b8b1

File: src/main/java/creativei/enums/CaseIndex.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/enums/CaseIndex.java]: added in commit fb5477c

File: src/main/java/creativei/enums/CaseIndex.java [hunk 4]
```diff
@@ -18,4 +31,28 @@ public int getIndex() {
...
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+    static List<String> list = new ArrayList<>();
+    static {
+        for (CaseIndex caseIndex : CaseIndex.values()) {
+            try {
+                list.add(new ObjectMapper().writeValueAsString(caseIndex));
```
Comment (aayushKhandpur) [inline src/main/java/creativei/enums/CaseIndex.java]: Should avoid creating ObjectMapper in loop

**Review by aayushKhandpur — APPROVED** [review]

File: src/main/java/creativei/enums/CaseIndex.java (hunk 4 above)
Comment (jitendrajaria) [reply src/main/java/creativei/enums/CaseIndex.java]: Resolve in commit 9c42295

**Pull request merged**